store/rootmultistore.go

```diff
@@ -2,10 +2,13 @@ package store
 
 import (
 	"fmt"
+	"strings"
 
+	"golang.org/x/crypto/ripemd160"
+
```

Comment:
No line between external imports.

Reply:
https://github.com/tendermint/coding/blob/master/go/coding_standard.md#importing-libraries

In some cases, it's nice to separate into three: standard lib, external libs, tendermint libs.